acc_test/data/valid_with_resizable_pci_cluster.tf

```diff
@@ -16,8 +16,8 @@ resource "instaclustr_cluster" "resizable_pci_cluster" {
         name = "AWS_VPC"
     }
     rack_allocation = {
-        number_of_racks = 3
-        nodes_per_rack = 1
+        number_of_racks = 4
```

Comment:
Why was the number of racks increased?

Reply:
Just wanted to check getting IPs out with an even number (or really just any number other than 3 nodes)